flashcards/backend/decks/factories.py

```diff
@@ -0,0 +1,76 @@
+from factory import Faker, Sequence, SubFactory
+from factory.django import DjangoModelFactory, ImageField
+from .models import Category, Deck, Flashcard, Tag, DifficultyLevel
```

Comment:
PEP8 missing new line after lib imports
I'd consider absolute import in line 3

Reply:
ok, now it's updated